lib/sorbet-rails/model_rbi_formatter.rb

```diff
@@ -74,6 +74,7 @@ def generate_base_rbi(root)
       self.model_relation_class_name,
       superclass: "ActiveRecord::Relation",
     )
+    model_relation_rbi.create_include(self.model_query_methods_returning_relation_module_name)
     model_relation_rbi.create_extend("T::Sig")
```

Comment:
While you're at it, can you remove these `extend T::Sig` lines? based on @paracycle we don't need them in RBI files